Buildings/ThermalZones/EnergyPlus/UsersGuide.mo

```diff
@@ -33,23 +33,20 @@ The <code>.epw</code> file will be used by the EnergyPlus envelope model, and th
 file will be used by the Modelica model, and must be specified by the parameter <code>weaName</code>
 in the instance <code>building</code>.
 </li>
+</ol>
+The following coupling objects can then be integrated in the model that contains the instance 
+<code>building</code>, or in any model instantiated by that model
+<ul>
 <li>
-In the model that contains the instance <code>building</code>,
-or in any model instantiated by that model, instantiate
-for each thermal zone an instance of
```

Comment:
I was originally confused by "for each thermal zone", since it might suggest to some reader that you are required to instantiate a `ThermalZone` model for all the thermal zones in the IDF, when in fact you only need to do so for zones that you want to control (with the caveat that the other zones will be free floating).